### PR TITLE
added libmikmod

### PR DESCRIPTION
**Pull request by unxmaal**

### CODE REVIEW

**Comment by unxmaal** [comment]
Added original spec file.

File: packages/libmikmod/SPECS/libmikmod.spec [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+Version:        3.3.11.1
+Release:        7%{?dist}
+License:        GPLv2 and LGPLv2+
+URL:            http://mikmod.sourceforge.net/
+Source0:        http://downloads.sourceforge.net/mikmod/libmikmod-%{version}.tar.gz
+Patch0:         libmikmod-64bit.patch
+Patch1:         libmikmod-multilib.patch
+Patch2:         libmikmod-cflags.patch
+BuildRequires:  gcc
+BuildRequires:  alsa-lib-devel pulseaudio-libs-devel
```
Comment (danielhams) [inline packages/libmikmod/SPECS/libmikmod.spec]: We don't have alsa-lib-devel or pulseaudio-libs-devel on irix - so building will fail here. How are you building this so it works? The correct build that skips checks is `rpmbuild -ba PACKAGE.spec --nocheck` - please don't skip dependency checking.

File: packages/libmikmod/SPECS/libmikmod.spec [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+%description
+libmikmod is a library used by the mikmod MOD music file player for
+UNIX-like systems. Supported file formats include MOD, STM, S3M, MTM,
+XM, ULT and IT.
+
+%package devel
+Summary:        Header files and documentation for compiling mikmod applications
+Provides:       mikmod-devel = %{version}-%{release}
+#Requires:       %{name}%{?_isa} = %{version}-%{release}
```
Comment (danielhams) [inline packages/libmikmod/SPECS/libmikmod.spec]: This one should normally stay (uncommented) I think - it's a dependency on the regular package on the arch from the development package.

File: packages/libmikmod/libmikmod.spec.origfedora [hunk 3]
```diff
@@ -0,0 +1,249 @@
+Summary:        A MOD music file player library
```
Comment (danielhams) [inline packages/libmikmod/libmikmod.spec.origfedora]: General comment is that this `.spec.origfedora` is in the wrong place - it is currently packages/libmikmod/libmikmod.spec.origfedora and should be packages/libmikmod/SPECS/libmikmod.spec.origfedora

**Comment by unxmaal** [comment]
Should be fixed up now.

File: packages/libmikmod/libmikmod.spec.origfedora (hunk 3 above)
Comment (danielhams) [reply packages/libmikmod/libmikmod.spec.origfedora]: `.spec.origfedora` added - thanks - could you remove the one in the wrong place too pls. thanks.

**Comment by unxmaal** [comment]
Should be cleaned up again. I didn't understand what you meant the first time around.

**Comment by danielhams** [comment]
Thanks for flying rse airlines. We know you have a choice in airlines, and you chose rse.

**Pull request merged**